security/pkg/pki/ca/ca_test.go

```diff
@@ -141,6 +143,21 @@ func TestCreateSelfSignedIstioCAWithoutSecret(t *testing.T) {
 	if !signingCertFromSecret.Equal(signingCert) {
 		t.Error("CA signing cert does not match the K8s secret")
 	}
+
+	// Check the siging cert stored in K8s configmap.
+	cmc := configmap.NewController(caNamespace, client.CoreV1())
+	strCertFromConfigMap, err := cmc.GetCATLSRootCert()
+	if err != nil {
+		t.Errorf("Cannot get the CA cert from configmap (%v)", err)
+	}
+	cert, _, _, _ := ca.GetCAKeyCertBundle().GetAllPem()
```

Comment:
could you pls explain a bit what's the difference between ```bundle.certBytes``` and ```bundle.rootCertBytes``` ? can we just use ```rootCertBytes``` ?